cmd/openshift-install/gather.go

```diff
@@ -117,11 +118,15 @@ func runGatherBootstrapCmd(directory string) error {
 func logGatherBootstrap(bootstrap string, port int, masters []string, directory string) error {
 	logrus.Info("Pulling debug logs from the bootstrap machine")
 	client, err := ssh.NewClient("core", fmt.Sprintf("%s:%d", bootstrap, port), gatherBootstrapOpts.sshKeys)
-	if err != nil && len(gatherBootstrapOpts.sshKeys) == 0 {
-		return errors.Wrap(err, "failed to create SSH client, ensure the proper ssh key is in your keyring or specify with --key")
-	} else if err != nil {
+	if err != nil {
+		if errno, ok := err.(syscall.Errno); ok && errno == syscall.ECONNREFUSED {
```

Comment:
hmm.. shouldn't see https://golang.org/pkg/net/#OpError ? and the OpError.Err contains the syscall error?

Reply:
I'll just wait for Go 1.13 and [the `Unwrap` business][1].

[1]: https://blog.golang.org/go1.13-errors#TOC_3.1.